examples/step-6/doc/intro.dox

```diff
@@ -167,6 +175,49 @@ module that explains how these constraints can be computed and what classes in
 deal.II work on them.
 
 
+<h3> How to deal with hanging nodes in practice </h3>
+
+The practice of hanging node constraints is rather simpler than the
+theory we have outlined above. In reality, you will really only have to
+add about half a dozen lines of additional code to a program like step-4
+to make it work with adaptive meshes that have hanging nodes. The
+interesting part about this is that it is entirely independent of the
+equation you are solving: How the constraints look like has nothing
```

Comment:
'what the constraint looks like'

Reply:
I think it might be more descriptive to write 'the algebraic nature of constraints' or something similar, but thats just me.

Reply:
I'm going with your suggested wording.

Since you've marked this up for me in other contexts as well, can you explain (so I can learn) why "How these constraints look like..." is wrong and "What these constraints look like..." is correct?

Reply:
Its surprisingly difficult, as a native speaker, to explain this rule.

Since the question "How does it look like?" ends in "like", the answer must be a simile: "it looks like *a linear system*". In particular, we are comparing *it* (a (pro)noun) to another noun. Hence the question should start with "What" since the answer is a noun: starting with 'How" implies that the answer is an adverb or an adjective (e.g., "How did you run?" "I ran *quickly*" or "How does it look?" "It looks *good*") which is not the case.

Since questions ending in *like* must be answered as similes, I don't think its possible to ever start such a question (with correct grammar, of course) with anything but *What*: i.e., the answer must be formatted as "*noun* looks like *noun*", so using anything but nouns or pronouns won't work.